index.bs

```diff
@@ -2879,6 +2864,7 @@ partial interface MLGraphBuilder {
         1. If the sequence length of |newShape| is not equal to the [=rank=] of |inputDesc|, then [=exception/throw=] a "{{DataError}}" {{DOMException}}.
         1. Let |outputDesc| be a copy of |inputDesc|.
         1. [=list/For each=] |index| in [=the range=] 0 to the [=rank=] of |input|, exclusive:
+            Issue: Can this be replaced with [=unidirectionally broadcasting the shapes=] |inputDesc| and |newShape|.
```

Comment:
Because it checks size of new shape is equal to input rank, I suppose it can unidirectionally broadcast input shape to new shape.

Reply:
I pushed a second commit that makes this update. Please take a look?

Reply:
Appears to me this comment is resolved, and Ningxin already signed off for the rest. Given he is IIRC out \~Feb 10th to 24th for the lunar new year, I propose completing this, and having Ningxin ask for any smaller changes in a later delta.